config/routes.rb

```diff
@@ -4,6 +4,9 @@
   # Defines the root path route ("/")
   # root "articles#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :likes, only: [:create, :destroy]
+      resources :comments, only: [:new, :create]
```

Comment:
- [ ] Please add method and a a correct controller name into the file comments_controller.rb. Right now, it seems like this and it should contain methods to create a comment using the current_user method as you did in post_controller.
your comments_controller file contains: 
![image](https://github.com/uno36/blog-app-rails/assets/91486791/559723ec-0345-4357-90a6-16cecc3a1a79)